### ISSUE TITLE
Finetuning the embedding model

### ISSUE THREAD

**Issue by bipowerhcmcity** [pyannote/pyannote-audio#1414]
Dear Mr. Hervé Bredin, 

We have our own dataset for improving the accuracy for embedding models, however, I saw in your Git that: the embedding model code is just used for the evaluation step. Is it correct or not? 

I'm finding out the way to finetune the embedding model, so I wonder how can I finetune the pretrained embedding model on the new dataset.
Thank you very much
Khoa
